.github/workflows/pr-preview.yml

```diff
@@ -42,14 +43,20 @@ jobs:
         run: |
           sudo apt install rsync
           BUILD_AS_FAST_AS_POSSIBLE=1 make package
+      - name: Store Whl File
+        uses: actions/upload-artifact@v3
+        with:
+          name: whl_file
```

Comment:
I think it is better here to use a valid wheel file name, e.g.: `streamlit-11.11.11-py2.py3-none-any.whl`. Otherwise, I think we won't be able to link it in a requirements.txt.

Reply:
Sort of a misnomer, but upload-artifact's `name:` field creates the name of the artifact folder, not the uploaded item itself. So folder is `whl_file` and contains wheel file with valid name 👍🏼 

<img width="400" alt="Screen Shot 2022-10-14 at 10 21 02 AM" src="https://user-images.githubusercontent.com/63436329/195905214-7cfde1cd-3793-49a0-a32c-9d480c0a4dbb.png">
<img width="300" alt="Screen Shot 2022-10-14 at 10 17 14 AM" src="https://user-images.githubusercontent.com/63436329/195905041-04c72780-b8a1-4e10-a0cb-11361c79eca3.png">

Reply:
Ok, I see 👍 There isn't any way to upload single files instead of a directory? This way it is not possible to install this wheel file from URL (which makes it less useful :().

Reply:
I can get the `whl_file` zip to include only the `.whl` file, but it doesn't seem there is a way around the zip to be able to upload/download just the file 😞 (note from actions/upload-artifacts doc):

However, when a workflow run finishes and an artifact is downloaded
 from either the UI or through the [download api](https://developer.github.com/v3/actions/artifacts/#download-an-artifact), a zip is 
dynamically created with all the file contents that were uploaded. 
There is currently no way to download artifacts after a workflow run finishes
 in a format other than a zip or to download artifact contents individually.